ts/input/tex/gensymb/GensymbConfiguration.ts

```diff
@@ -23,33 +23,42 @@
  */
 
 import {Configuration} from '../Configuration.js';
-import ParseMethods from '../ParseMethods.js';
+import {Symbol} from '../Symbol.js';
 import {TexConstant} from '../TexConstants.js';
 import {CharacterMap} from '../SymbolMap.js';
+import TexParser from '../TexParser.js';
 
 
 /**
- * Ohm symbol as in gensymb. Usually upright, but can be affected by fonts.
+ * Handle characters that are known units.
+ * @param {TexParser} parser The current tex parser.
+ * @param {Symbol} mchar The parsed symbol.
  */
-new CharacterMap('gensymb-ohm', ParseMethods.mathchar7, {
-  ohm:            '\u03A9'
-});
+function mathcharUnit(parser: TexParser, mchar: Symbol) {
+  console.log(mchar.attributes);
```

Comment:
Don't forget to remove this log command.

Reply:
Good catch. Thanks.
Note that I've left `\micro` as a unit now.

Reply:
> Note that I've left `\micro` as a unit now.

Understood.